src/main.rs

```diff
@@ -98,8 +101,18 @@ fn main() -> Result<(), Box<dyn Error>> {
         }
     };
     let path = matches.opt_str("output");
-    let commits_path = &matches.free[1];
-    let commits = fs::read_to_string(commits_path).unwrap();
+    let commits = if matches.opt_present("git") {
+        let process = std::process::Command::new("git")
+            .arg("log")
+            .arg("--numstat")
+            .output()?;
+        String::from_utf8(process.stdout)?
+    } else if matches.opt_present("input") {
+        let path = matches.opt_str("input").unwrap();
+        std::fs::read_to_string(path)?
+    } else {
+        String::new()
```

Comment:
Hmm, while this actually runs fine, maybe something like
```rust
todo!("Reading git log from stdin")
```
or `unimplemented!`.

While this would exit the program, it would also make clear, both to the user, and the programmer, that here is something missing. And since this is only temporary, it should be fine imo.

Reply:
:wrench: :heavy_check_mark: